scripts/mujoco/create_dataset.py

```diff
@@ -0,0 +1,115 @@
+import gymnasium as gym
+import minari
+from huggingface_sb3 import load_from_hub
+from stable_baselines3 import PPO, SAC, TD3
+from tqdm import tqdm
+
+ENV_IDS = [
+    ("InvertedPendulum", ("medium", "expert"), 100_000, "SAC"),
+    ("InvertedDoublePendulum", ("medium", "expert"), 100_000, "SAC"),
+    ("Reacher", ("medium", "expert"), 500_000, "SAC"),
+    ("Pusher", ("medium", "expert"), 500_000, "SAC"),
+    ("HalfCheetah", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Hopper", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Walker2d", ("simple", "medium", "expert"), 1_000_000, "SAC"),
+    ("Swimmer", ("expert"), 1_000_000, "SAC"),
+    ("Ant", ("simple", "medium"), 1_000_000, "SAC"),
+    ("Humanoid", ("simple", "medium", "expert"), 1_000_000, "SAC"),
```

Comment:
how did you choose the number of steps? Why they differ from each other?

Reply:
One million was chosen as the base dataset length.

The pendulum environment datasets were set to 100,000 because the environments are so simple.

And the manipulator dataset length was set to 500,000, Because they're a bit more complex.